.eslintignore

```diff
@@ -5,3 +5,5 @@ singleplayer/examples
 packages/rune-games-cli/app/
 packages/rune-games-cli/__temp/
 packages/rune-games-cli/src/generated/
+packages/rune-games-cli/cjs/
+multiplayer/examples/sudoku/
```

Comment:
I added the whole `multiplayer/examples` folder to ignore in my outmatched PR, but maybe this is better.

Reply:
I actually think `multiplayer/examples/` is better, I changed it.